list_files/master_son.list

```diff
@@ -1778,8 +1778,9 @@ LA LA3194 LA173 US71/1
 
 # Maine (ME)
 # ---------------------
-ME ME5 US1_N UniAve
-ME ME9 US1_Sca ScaConn
+ME ME5 UniAve US1_N
+ME ME9 US1_Sca ScaCon
```

Comment:
I looked at your Maine map a while back, and saw that the [ScaCon](https://travelmapping.net/hb/showroute.php?units=miles&u=master_son&r=me.scacon) itself is a suspicious gap in your travels.